### PR TITLE
Implement initial README for the learn section

### PR DESCRIPTION
**Pull request by zackproser**
- Separation of this repository into docs/ (Engineering-owned) and learn/ (DevRel-owned) is imminent
- Add information to help orient newcomers to the examples and how they work
- Add detailed setup and running instructions for Google Colaboratory

## Problem

We'd like to have a README for the learn section to greet and orient our users and community members.

## Solution

(For ease of review, please see the rendered README [here](https://github.com/pinecone-io/examples/blob/d9fecc568eaa9c1d0182026bac6c79c6ccf65b6f/learn/README.md)).

I added some light introduction content, and then focused on step by step instructions for getting the examples running in Google Colaboratory. 

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Infrastructure change (CI configs, etc)
- [x] Non-code change (docs, etc)
- [ ] None of the above: (explain here)

## Test Plan

I verified my instructions by following them to run example notebooks in Google Colab successfully.

### CODE REVIEW

**Comment by zackproser** [comment]
Realizing now I'll need to swap in a Pinecone text logo with a solid background to work for both light and dark mode folks!

**Comment by gkogan** [comment]
https://www.pinecone.io/newsroom/

File: learn/README.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+Pinecone.io offers a generous free tier that's ideal for learning and experimenting, and even building applications that leverage vector databases.
+
+Head over to [the Pinecone.io dashboard](https://app.pinecone.io/) and create a new account, which you can do in under a minute. 
+
+## Step 2. Get your Pinecone.io API key
+
+On the left-hand side of the Pinecone dashboard, click the API Keys header. You'll have a default API key already available. **_Your API Key is a secret - don't share it or commit it to version control_!**
+
+![Get your Pinecone.io API key](./images/pinecone-api-keys.png)
```
Comment (jamescalam) [inline learn/README.md:33]: we should add a mention for "Remember to take note of your environment too" as users will need that just as much as their API key (and their conveniently located in the same place)

File: learn/README.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+While logged into a Google account, visit [Google Colaboratory](https://colab.research.google.com/). This is a Google service which allows you to run Jupyter Notebooks on Google infrastructure to greatly simplify experimenting, exploring and sharing your results 
+without needing to worry about creating or managing cloud compute environments.
+
+## Step 4. Perform the one-time setup for Google Colaboratory 
+
+You will only need to complete these next steps one time, and can then skip them in the future when running additional example Notebooks.
+
+Start by clicking File > Open Notebook in the top left menu of Google Colab and be sure to click the GitHub tab. You can then either: 
+
+1. Paste the full URL (including the `.ipynb` extension) into the URL bar in the modal, or you can enter `pinecone-io` (which is the Pinecone.io GitHub organization name) and then enter `pinecone-io/examples` in the repository field:
```
Comment (jamescalam) [inline learn/README.md:46]: people can also just click the "Colab" button at the top of most notebooks (but not all, so these instructions are valid for where we might be missing that — but we should aim to have colab shields in all notebooks to minimize friction)

File: learn/README.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+With those steps complete, you can re-try opening your desired Notebook and it should load correctly: 
+
+![A correctly loaded Jupyter Notebook in Google Colab](./images/google-colab-loaded-jupyter-notebook.png)
+
+### Step 4. Securely set your Pinecone API key 
+
+As a best practice, you should never hardcode credentials such as an API key into any source code, such as a Jupyter Notebook, because it's easy to accidentally share or commit the key unintentionally, leading to compromise of your account and its resources. 
+
+The Pinecone.io example Jupyter Notebooks expect a Pinecone.io API key available in the `PINECONE_API_KEY` environment variable.
```
Comment (jamescalam) [inline learn/README.md:74]: and an environment var

**Review by jamescalam — CHANGES_REQUESTED** [review]
couple of comments but looks great!

File: learn/README.md (hunk 2 above)
Comment (zackproser) [reply learn/README.md:46]: Good call and will add - one thing I noticed yesterday toward EOD is that some of the Colab buttons 404'd / didn't load properly, but I'll need to dig in a bit more to figure out what's going on.

File: learn/README.md (hunk 3 above)
Comment (zackproser) [reply learn/README.md]: Added in [b82957b](https://github.com/pinecone-io/examples/pull/240/commits/b82957b861fe9916f717d96d066fd0ab9f4cb5f6)

File: learn/README.md (hunk 2 above)
Comment (zackproser) [reply learn/README.md]: Added in [b82957b](https://github.com/pinecone-io/examples/pull/240/commits/b82957b861fe9916f717d96d066fd0ab9f4cb5f6)

File: learn/README.md (hunk 1 above)
Comment (zackproser) [reply learn/README.md]: Good catch - added in [b82957b](https://github.com/pinecone-io/examples/pull/240/commits/b82957b861fe9916f717d96d066fd0ab9f4cb5f6)

**Comment by zackproser** [comment]
Alright - this one should be ready for another look!

**Comment by gkogan** [comment]
@zackproser "Production-ready" doesn't need to be capitalized. LGTM! Let's also add a repo description -- the one that displays in top-right.

**Comment by zackproser** [comment]
> @zackproser "Production-ready" doesn't need to be capitalized. LGTM! Let's also add a repo description -- the one that displays in top-right.

@gkogan Roger, thanks! I've pushed that fix in my fork's branch [here](https://github.com/zackproser/examples/commit/84acbd833bf34e2fd4c4c3bbf7f562b1d2273494) but it hasn't yet synced / shown up here yet :/

**Comment by zackproser** [comment]
Thanks for reviews! :bow:

**Pull request merged**